GitUI/CommandsDialogs/FormCreateBranch.cs

```diff
@@ -34,9 +35,9 @@ public FormCreateBranch(GitUICommands commands, ObjectId? objectId, string? newB
         {
             InitializeComponent();
 
-            // work-around the designer bug that can't add controls to FlowLayoutPanel
-            ControlsPanel.Controls.Add(Ok);
-            AcceptButton = Ok;
```

Comment:
I fixed this bug in VS 17.2 (I think)... I'll fix other instances of this bug in follow up PRs.